ChangeLog

```diff
@@ -1,5 +1,15 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x released xxxx-xx-xx
+
+Changes
+   * Added config.h option MBEDTLS_NO_UDBL_DIVISION, to prevent the use of
+     64-bit division.
+
```

Comment:
Hi @hanno-arm. What you may not know, is we're fixing this for a customer who asked us specifically to remove support for division with `mbedtls_t_udbl`. See bug #708. We have taken a very strange path to achieve that - but that's where we finally arrived. In the context of that `MBEDTLS_NO_UDBL_DIVISION` makes sense as it's what they asked for.

Generally, we prefer to say through the configurations what we do - not what we don't. So `MBEDTLS_HAVE_INT32` or `MBEDTLS_HAVE_INT64` make sense. Saying `MBEDTLS_NO_INT64` less so. So I take your point in terms of what we're functionally introducing - but I don't want to introduce the logical opposite of `MBEDTLS_HAVE_UDBL` with `MBEDTLS_NO_UDBL`.